_consulting/architecting.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: Cloud Architecting
+layout: splash
+permalink: /consulting/architecting
+
+header:
+  overlay_image: /assets/images/website/architecting.jpg
+  # image: /assets/images/website/architecting.jpg
+  teaser: /assets/images/website/architecting.jpg
```

Comment:
this behaves differently than the one above. Is there a preferred use of image, teaser, and/or overlay?